build_image.sh

```diff
@@ -202,12 +202,12 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     zip -g $OUTPUT_ABOOT_IMAGE .platforms_asic
 
     if [ "$ENABLE_FIPS" = "y" ]; then
-        echo "sonic_fips=1" > kernel-cmdline
+        echo "sonic_fips=1" >> kernel-cmdline-append
```

Comment:
Do you want to empty the file "kernel-cmdline-append" before the first usage?

Reply:
It's indeed better if the file is reset before starting appending to it to avoid duplicated content.
I addressed your comment here https://github.com/sonic-net/sonic-buildimage/pull/15343
This PR will need to be cherry-picked to 202205 where this change also merged.